lms/djangoapps/api_manager/users/views.py

```diff
@@ -1012,6 +1009,13 @@ def get(self, request, user_id, course_id):
         if len(queryset):
             current_grade = queryset[0].grade
             proforma_grade = queryset[0].proforma_grade
+            progress_summary = json.loads(queryset[0].progress_summary)
+            grade_summary = json.loads(queryset[0].grade_summary)
+            grading_policy = json.loads(queryset[0].grading_policy)
+        else:
+            progress_summary = grades.progress_summary(student, request, course_descriptor)  # pylint: disable=W0612
```

Comment:
Yes, that is correct -- please use the messages instead of the codes for all pylint instructions